DocumentService/Controllers/DocumentsController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace DocumentService.Controllers
+{
+    [Route("api/")]
+    [ApiController]
+    public class DocumentsController : ControllerBase
+    {
+
+        public DocumentsController()
+        {
+
+        }
+
+        /// <summary>
+        /// Upload a document.
+        /// </summary>
+        /// <param name="CorrelationId">Correlation identifier of the operation.</param>
+        /// <param name="UserName">Azure AD identifier of the user uploading the document.</param>
+        /// <param name="FileName">File name of the document.</param>
+        /// <param name="ShortDescription">Short description of the document.</param>
+        /// <param name="SubmissionMethod">Indicates how the file was submitted to Transport Canada. (“FAX”, “MAIL”, “EMAIL”).</param>
```

Comment:
I'm not sure I like it since it is specific to our product but Jon wants us to add "EMER" as a submission method as well. So I guess let`s include it for now so we don`t lose sight of it and if needed we can refactor later on.